moxin-frontend/src/shared/widgets/c_button.rs

```diff
@@ -0,0 +1,92 @@
+use makepad_widgets::*;
+
```

Comment:
It took me a second to understand that `widgets.rs` has styling of existing widgets (no new widgets), and the new widgets folder has actual widgets.
I think we should either put new things like `c_button` in the root of `shared` module as we're already doing, or create a new submodule widgets (like you did now, and move all shared widgets there) but also rename `widgets.rs` or delete it and move its contents into styles.rs